src/Carousel.js

```diff
@@ -269,6 +288,7 @@ class Carousel extends React.Component {
           onClick={this.handlePrev}
         >
           {prevIcon}
+          {prevLabelNode}
```

Comment:
I did it on purpose, because I cannot see an useful usage of empty labels for screen readers. They'll be unable to read them!

That's also why I added a test on this.